spec/models/label_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe Label, type: :model do
+  before do
+    @user = FactoryBot.create(:user)
```

Comment:
`FactoryBot.`という記載は設定ファイルに追記を行うことで、省略することもできます。
```suggestion
    @user = create(:user)
```